public/stylesheets/chat_room_style.css

```diff
@@ -55,6 +55,18 @@
   float: right;
 }
 
+img.conversation-avatar-received {
+  height: 80px;
+  width: 80px;
+  float: left;
+}
+
+img.conversation-avatar-sent {
+  height: 80px;
+  width: 80px;
+  float: right
+}
+
```

Comment:
We can just use `conversation-avatar` and reuse the `sent` etc.

Reply:
but then how do we differentiate receive and sent

Reply:
Here https://github.com/ZijiaZhang/CPSC436_Project/blob/11d79325f104bd56d6f08319005fcef1fde8b88c/public/stylesheets/chat_room_style.css#L44

Reply:
better keep as what I wrote before, change .sent will not work